build.gradle

```diff
@@ -394,6 +394,8 @@ dependencies {
     pitest 'org.pitest:pitest-junit5-plugin:0.12'
 
     testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: versions.junit
+    testImplementation group: 'org.mockito', name: 'mockito-core', version: '3.4.6'
+    testImplementation group: 'org.mockito', name: 'mockito-inline', version: '3.6.28'
```

Comment:
why we need these dependencies to include  as we have already spring-boot-starter-test in project. Please clarify it.

Reply:
Latest version of mockito provides methods to mock static methods which was used for condition coverage.